fork.html

```diff
@@ -0,0 +1,170 @@
+<html>
+  <head>
+    <title>The OpenTF fork preview is now available!</title>
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no" />
+    <meta name="robots" content="index" />
+
+    <meta name="description" content="The OpenTF Foundation. Supporting an impartial, open, and community-driven Terraform." />
+
+    <link rel="canonical" href="/announcement" />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Montserrat:wght@400;500;600&display=swap" />
+    <link rel="stylesheet" type="text/css" href="../style.css">
+    <link rel="apple-touch-icon" sizes="180x180" href="favicons/apple-touch-icon.png">
+    <link rel="icon" type="image/png" sizes="16x16" href="favicons/favicon-16x16.png">
+    <link rel="icon" type="image/png" sizes="32x32" href="favicons/favicon-32x32.png">
+    <link rel="icon" type="image/png" sizes="192x192" href="favicons/android-chrome-192x192.png">
+    <link rel="icon" type="image/png" sizes="512x512" href="favicons/android-chrome-512x512.png">
+    <link rel="manifest" href="favicons/site.webmanifest">
+    <meta name="msapplication-TileColor" content="#da532c">
+    <meta name="theme-color" content="#ffffff">
+
+    <meta property="twitter:title" content="The OpenTF fork preview is now available!">
+    <meta property="twitter:description" content="Learn more about the fork and our roadmap at https://opentf.org/fork">
+    <meta property="twitter:image" content="https://opentf.org/images/og.png">
+    <meta property="twitter:url" content="https://opentf.org/fork">
+    <meta property="twitter:card" content="summary_large_image">
+
+    <meta property="og:title" content="The OpenTF fork preview is now available!">
+    <meta property="og:description" content="Learn more about the fork and our roadmap at https://opentf.org/fork">
+    <meta property="og:image" content="https://opentf.org/images/og.png">
+    <meta property="og:url" content="https://opentf.org/fork">
+    <meta property="og:type" content="website">
+    
+    <script async defer src="https://buttons.github.io/buttons.js"></script>
+  </head>
+  <body>
+    <div class="container mx-auto">
+      <a href="/">
+          <picture>
+              <source srcset="/images/on-dark%402x.png" media="(prefers-color-scheme: dark)">
+              <img width="342px" height="200px" src="/images/on-light%402x.png" class="logo mx-auto" alt="OpenTF Foundation">
+          </picture>
+      </a>
+
+      <section id="announcements">
+        <h1 class="text-left">The OpenTF fork preview is now available!</h1>
```

Comment:
The OpenTF Repository is Now Available

Reply:
I left it as "The OpenTF fork is now available!" I think at this early stage, we should still call it the "fork" in the headline, as that is what people are expecting, and will probably resonate better. Elsewhere in the post, I refer to it as "repo." Future announcements can talk more about it as a repo.